package.json

```diff
@@ -56,18 +55,19 @@
 		"build-docker": "node bin/build-docker.js",
 		"build-static": "npx ncp static public",
 		"prebuild-server": "mkdirp build",
-		"build-server": "cross-env-shell BROWSERSLIST_ENV=server NODE_PATH=$NODE_PATH:server:client:. webpack --display errors-only --config client/webpack.config.node.js",
+		"build-server": "cross-env-shell BROWSERSLIST_ENV=server webpack --display errors-only --config client/webpack.config.node.js",
 		"build-client": "npm run build-client-evergreen",
-		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only && npm run validate-fallback-es5",
-		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
+		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
+		"postbuild-client-fallback": "npm run validate-fallback-es5",
+		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
 		"build-client-both": "CONCURRENT_BUILDS=2 concurrently -c cyan -n \"fallback ,evergreen\" \"npm run build-client-fallback\" \"npm run build-client-evergreen\"",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || cross-env-shell NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} npm run build-client-both",
```

Comment:
I know this code was already here, but why do some scripts (like this one) extend `NODE_ARGS`, while others (like `preanalyze-bundles` and `prewhybundled`) rewrite it?

Reply:
> why do some scripts extend `NODE_ARGS`, while others rewrite it?

That's a good point and I'm cleaning up `NODE_ARGS` in https://github.com/Automattic/wp-calypso/pull/38931/commits/42fb4f6fd40edb184a89e7ad090d805dd9b1e508 and further consolidating the build scripts in https://github.com/Automattic/wp-calypso/pull/38931/commits/1d9918436debe6fed1b4b062594a4b37e366c14d


After these changes, the scripts set and use the `NODE_ARGS` variable only at the place where `node` is actually spawned to run webpack or the server bundle -- `build-client-do` and `start-bundle-do`, respectively.

No other tasks set `max_old_space_size` now. The default value is OK for everyone, and there's also a way (used by `wp-desktop`) to override it by setting the `NODE_ARGS` environment variable.

The `build-client-do` consolidation works as follows: instead of two NPM tasks:
```
"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=fallback node bin/webpack [lot of webpack and node arguments]"
"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen node bin/webpack [lot of webpack and node arguments]"
```
there are three and duplication is removed:
```
"build-client-do": "node bin/webpack [lot of webpack and node arguments]"
"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=fallback npm run build-client-do"
"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen npm run build-client-do"
```

Reply:
That's a great solution, thank you! 👍

Reply:
I'd like to try out the `NODE_OPTIONS` environment variable, too. It's supported directly by Node.js, and was added only in 8.x, which is not that ancient. We probably started setting `max_old_space_size` at a time when `NODE_OPTIONS` wasn't available yet.